.github/dependabot.yml

```
@@ -9,7 +9,19 @@ updates:
       security:
         applies-to: security-updates
         patterns: ["*"]
+  - package-ecosystem: "gomod"
+    directory: "/map-generator"
+    schedule:
+      interval: "monthly"
+    open-pull-requests-limit: 0 # exclude version updates
+    groups:
+      golang:
+        applies-to: security-updates
+    groups:
```

YAMLlint (1.38.0)

[error] duplication of key "groups" in mapping

(key-duplicates)